src/components/map-overlays/map-overlays.component.tsx

```diff
@@ -154,7 +159,7 @@ const MapOverlays = ({
                 location={poi.location}
                 text={poi.displayName}
                 onPress={() => {
-                  setMarkerLocation(poi);
+                  setPOIMarkerLocation(poi);
```

Comment:
Might be better to rename this to `onPOIMarkerPress`, to make it clear what event is being handled by the function. Also, because the function doesn't just set the marker location anymore, so its name is misleading.

Reply:
You are right, modified it to `onPOIMarkerPress`, see in commit 8804195

Reply:
Thanks!